packages/reason-relay/src/ReasonRelay.re

```diff
@@ -1142,34 +1146,34 @@ type optimisticUpdaterFn = RecordSourceSelectorProxy.t => unit;
 
 type mutationError = {message: string};
 
-type useMutationConfig('response, 'variables) = {
+type useMutationConfig('response, 'rawResponse, 'variables) = {
   onError: option(mutationError => unit),
   onCompleted: option(('response, option(array(mutationError))) => unit),
   onUnsubscribe: option(unit => unit),
-  optimisticResponse: option('response),
+  optimisticResponse: option('rawResponse),
   optimisticUpdater: option(optimisticUpdaterFn),
   updater: option(updaterFn('response)),
   variables: 'variables,
 };
 
-type _useMutationConfig('response, 'variables) = {
+type _useMutationConfig('response, 'rawResponse, 'variables) = {
```

Comment:
not really new, but we could use a clearer naming convention for types used only for bindings, like `myTypeRaw` or `myTypeJs`

Reply:
Yup, agree! Probably best as a separate PR though.